### PR TITLE
any and max

### PR DESCRIPTION
**Pull request by KevinUnadkat**
Added any from utility to paddle backend and updated Update statistical by adding max

### CODE REVIEW

**Comment by KevinUnadkat** [comment]
issue - any #10747

**Comment by MahmoudAshraf97** [comment]
Please change the torch specific function calls to paddle such as `paddle.as_tensor `to `paddle.to_tensor` and make sure to test the function on a paddle or ivy array first

**Comment by KevinUnadkat** [comment]
should i change for both "all" and "any"

**Comment by MahmoudAshraf97** [comment]
I already fixed all, so you can change any only

**Comment by KevinUnadkat** [comment]
@MahmoudAshraf97 
did the changes

File: ivy/functional/backends/paddle/utility.py [hunk 1]
```diff
@@ -37,6 +37,22 @@ def any(
     keepdims: bool = False,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    x = paddle.to_tensor(x).type(paddle.bool)
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/utility.py:40]: paddle tensors don't have `.type` method, pass the type to the `dtype` argument of `.to_tensor` instead

File: ivy/functional/backends/paddle/utility.py [hunk 2]
```diff
@@ -37,6 +37,22 @@ def any(
...
+    if axis is None:
+        num_dims = len(x.shape)
+        axis = list(range(num_dims))
+    if isinstance(axis, int):
+        return paddle.any(x, dim=axis, keepdim=keepdims, out=out)
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/utility.py:45]: `paddle.any` doesn't have `out` argument

File: ivy/functional/backends/paddle/utility.py [hunk 3]
```diff
@@ -37,6 +37,22 @@ def any(
...
+    dims = len(x.shape)
+    axis = [i % dims for i in axis]
+    axis.sort()
+    for i, a in enumerate(axis):
+        x = paddle.any(x, dim=a if keepdims else a - i, keepdim=keepdims, out=out)
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/utility.py:50]: Same as L45

**Review by MahmoudAshraf97 — CHANGES_REQUESTED** [review]

**Comment by KevinUnadkat** [comment]
@MahmoudAshraf97 
resolved the issues

**Review by MahmoudAshraf97 — CHANGES_REQUESTED** [review]

**Comment by KevinUnadkat** [comment]
@MahmoudAshraf97 
done

**Comment by KevinUnadkat** [comment]
@MahmoudAshraf97 
 can u also review max from statistical i accidently committed in same pr .

**Pull request merged**